test/benchmark/api_v1_1_tests.py

```diff
@@ -0,0 +1,137 @@
+import time
+
+from proboscis.asserts import assert_equal
+from proboscis.asserts import assert_false
+from proboscis import SkipTest
+from proboscis import test
+from proboscis import before_class
+from proboscis import after_class
+from json import loads
+
+from modules.logger import Log
+from modules.amqp import AMQPWorker
+from modules.worker import WorkerThread, WorkerTasks
+from config.api1_1_config import *
+from config.amqp import *
+from on_http_api1_1 import NodesApi as Nodes
+from on_http_api1_1 import WorkflowApi as Workflows
+from tests.api.v1_1.discovery_tests import DiscoveryTests
+from tests.api.v1_1.poller_tests import PollerTests
+
+from benchmark import ansible_ctl
+from benchmark.utils import parser
+from benchmark.utils.case_recorder import caseRecorder
+
+LOG = Log(__name__)
+
+class BenchmarkTests(object):
+    def __init__(self, name):
+        ansible_ctl.render_case_name(name)
+        self.__data_path = ansible_ctl.get_data_path_per_case()
+        self.case_recorder = caseRecorder(self.__data_path)
+        self.client = config.api_client
+        self.node_count = 0
+
+    def prepare_case_env(self):
```

Comment:
Best practice: 
Looks like `BenchmarkTests` is the base class. The real test case class will extend it , right ?
So the base function which is for inherit using need to be start with single underscore '_' to imply it's `protected` function